packages/crypto/src/crypto.ts

```diff
@@ -97,7 +170,7 @@ export const hpkeDecrypt = ({
 };
 
 /**
- * Decrypt an encrypted credential bundle.
+ * Decrypt an encrypted email auth/recovery credential bundle.
```

Comment:
maybe a slight rewording: `Decrypt an encrypted credential bundle, i.e. for use with Turnkey Email Auth + Recovery.`

Reply:
ehh, this function specfically works with email bundles, since it's a string. Our export bundles are a bit different, I should probably include a helper function for that as well